### PR TITLE
README refactoring

### PR DESCRIPTION
**Pull request by WieFel**

### CODE REVIEW

**Review by calvintam236 — CHANGES_REQUESTED** [review]
Import instruction is outdated

**Comment by WieFel** [comment]
@calvintam236 just noticed that

**Review by calvintam236 — COMMENTED** [review]
L13 syntax incorrect

**Comment by calvintam236** [comment]
Also some of the ``` has 4`

File: README.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
-# tinycolor
+# tinycolor2
 > TinyColor is a small library for Flutter color manipulation and conversion
 
 A port of [tinycolor2](https://github.com/bgrins/TinyColor) by [Brian Grinstead](https://github.com/bgrins)
```
Comment (calvintam236) [inline README.md:4]: Outdated url?

File: README.md [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
+A `TinyColor` receives a `Color` as parameter.
 
 ```dart
-import 'package:tinycolor/tinycolor.dart';
+import 'package:tinycolor2/tinycolor2.dart';
 
-final TinyColor = TinyColor(Colors.green);
+final TinyColor tinyColor = TinyColor(Colors.green);
 ```
 Now you can also use the package to extend the native Color class with all the same features, but simpler. To use extension update, make sure to change envieronment sdk version in pubspec like this: ` sdk: ">=2.6.0 <3.0.0"`
```
Comment (calvintam236) [inline README.md:15]: `Color`

**Comment by calvintam236** [comment]
Also missing `.toHex8()` and `.equals()/operator==`

File: README.md (hunk 1 above)
Comment (WieFel) [reply README.md:4]: Which one. They work fine for me.

**Comment by WieFel** [comment]
> Also missing `.toHex8()` and `.equals()/operator==`

@calvintam236 please go ahead and add it as you want it

File: README.md (hunk 1 above)
Comment (calvintam236) [reply README.md:4]: First one should be this repo url? You can mention the original repo.

File: README.md (hunk 2 above)
Comment (calvintam236) [reply README.md:15]: Not that line. The Color in the text is not quoted.

File: README.md (hunk 1 above)
Comment (WieFel) [reply README.md:4]: Why? It's referencing the original repo https://github.com/bgrins/TinyColor and author https://github.com/bgrins, which is correct in my opinion.
We could rather additionally credit the repo/authors we forked this from!?

File: README.md (hunk 2 above)
Comment (WieFel) [reply README.md:15]: oh sorry

File: README.md [hunk 3]
```diff
@@ -240,7 +242,7 @@ Colors.red.compliment;
 
 ### mix
 
-`mix: function(toColor, amount = 10) -> TinyColor`. Blends the color with another color a given amount, from 0 - 100, default 50.
+`mix: function(toColor, amount = 10) -> TinyColor`. Blends the color with another color a given amount, from `0` to `100`, default `50`.
```
Comment (calvintam236) [inline README.md:245]: should this default `10` per code?

Comment (WieFel) [reply README.md:245]: yes

**Review by calvintam236 — APPROVED** [review]

**Pull request merged**